.github/workflows/frontend-build-check.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         working-directory: frontend
         # TODO Reenable when https://github.com/webrecorder/browsertrix-cloud/issues/1618 is addressed
         # run: yarn format:check
-        run: yarn prettier --list-different .
+        run: echo "yarn format:check disabled"
```

Comment:
@emma-sg @SuaYoo assume this is intentional, right?

Reply:
Unsure... I think there was a tailwind class sorting issue where certain classes would be sorted differently in CI vs locally, but I'm not sure that's reason enough to totally disable this check

Reply:
Hm, maybe we revisit it later?